internal/modeldata.go

```diff
@@ -3,10 +3,9 @@ package internal
 import (
 	"context"
 	"fmt"
+	"github.com/qlik-oss/enigma-go"
```

Comment:
We should allign the version of go-fmt so that we don't get formatting changes for each change regarding dependencies